esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -82,6 +88,8 @@ const unsigned MAX_KEY_ROWS = 20;
 
 short days[12] = {31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
 
+
```

Comment:
very minor, but 2 new lines might not be needed

Reply:
agreed, I'll take a line out.